cegs_portal/search/templates/search/v1/partials/_experiment_index.html

```diff
@@ -0,0 +1,91 @@
+                <div class="flex justify-center items-center">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-icon h-4 w-4 mr-2 text-slate-500">
+                        <path d="M3.9 54.9C10.5 40.9 24.5 32 40 32H472c15.5 0 29.5 8.9 36.1 22.9s4.6 30.5-5.2 42.5L320 320.9V448c0 12.1-6.8 23.2-17.7 28.6s-23.8 4.3-33.5-3l-64-48c-8.1-6-12.8-15.5-12.8-25.6V320.9L9 97.3C-.7 85.4-2.8 68.8 3.9 54.9z"/>
+                    </svg>
+                    <div class="text-xl font-bold text-slate-500">Experiment Filters</div>
+                </div>
+                <div id="categorical-facets">
+                    {% for facet, values in facets.items %}
+                    <fieldset class="my-4" name="facetfield">
+                        <legend class="font-bold">{{ facet }}</legend>
+                        <div class="flex flex-row flex-wrap gap-1">
+                            {% for value in values %}
+                                <div class="ml-1">
+                                    <input id="{{ value.id }}" type="checkbox" name="{{ facet }}"/>
+                                    <label for="{{ value.id }}">{{ value.value }}</label>
+                                </div>
+                            {% endfor %}
+                        </div>
+                    </fieldset>
+                    {% endfor %}
+                </div>
+            </div>
+        </div>
+
+        <div id="experiment-list" class="basis-3/4">
+            {% for experiment in experiments %}
+            <a href="{% url 'search:experiment' experiment.accession_id %}" class="exp-list-content-container-link experiment-summary" data-accession="{{ experiment.accession_id }}" data-name="{{ experiment.name }}">
+                <div class="container items-center">
+                    <div class="flex justify-between">
+                        <div class="exp-name">{{ experiment.name }}</div>
+                    </div>
+                    <div>{{ experiment.description }}</div>
+                    <div class="flex justify-between">
+                        <div class="cell-lines">Cell Lines: {{ experiment.cell_lines }}</div>
+                        <div class="accession-id">{{ experiment.accession_id }}</div>
+                    </div>
+                </div>
+            </a>
+            {% endfor %}
+        </div>
+        </div>
+    </div>
+</div>
+</div>
+
+{% endblock %}
+
+{% block inline_javascript %}
+<script type="module">
+    import { facetFilterSetup } from "{% static 'search/js/experiment_list/facet_filter.js' %}";
+</script>
+<script>
+    function closeModal() {
+        let modal = document.getElementById("exp-modal");
+        modal.classList.add("closing") // Kicks off an animation on the modal dialog
+        modal.addEventListener("animationend", (event) => {
+            modal.remove(); // remove the dialog from the DOM once the animation is over
+        })
+    }
+</script>
```

Comment:
Do you need this here or in experiment.html? I feel like it should be only in one place.